{{ cookiecutter.project_slug }}/.github/workflows/ci.yml

```diff
@@ -72,4 +75,23 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         run: mkdocs build --strict
 {%- endif %}
+{%- elif cookiecutter.packaging == "uv" %}
+
+      - name: Install dependencies
+        run: uv sync
+
+      - name: Run mypy
+        run: uv run mypy .
+
+      - name: Run tests
+        run: uv run pytest
+{%- if cookiecutter.mkdocs %}
+
+      - name: Install doc dependencies
+        run: uv sync --group doc
+
+      - name: Check that documentation builds cleanly with MkDocs
+        if: matrix.os == 'ubuntu-latest'
+        run: uv run mkdocs build --strict
```

Comment:
I don't like that this block is repeated 3 times. Better to move it outside the `if cookiecutter.packaging` sections into a separate `if cookiecutter.mkdocs` section at the end